modules/csg/csg_shape.h

```diff
@@ -31,12 +31,12 @@
 #ifndef CSG_SHAPE_H
 #define CSG_SHAPE_H
 
-#define CSGJS_HEADER_ONLY
```

Comment:
This seemed unused, I can't find this define anywhere else in the codebase.

Reply:
Perhaps @reduz used https://github.com/dabroz/csgjs-cpp briefly?